src/app/components/Figure/Image/index.jsx

```diff
@@ -1,8 +1,26 @@
+import React from 'react';
 import styled from 'styled-components';
+import { number, string } from 'prop-types';
 
-const Image = styled.img`
+export const StyledImg = styled.img`
   display: block;
   width: 100%;
 `;
 
+export const Img = ({ alt, height, src, width }) => (
+  <StyledImg alt={alt} src={src} height={height} width={width} />
+);
+
+Img.propTypes = {
+  alt: string.isRequired,
+  height: number.isRequired,
+  src: string.isRequired,
+  width: number.isRequired,
+};
+
+// eslint-disable-next-line global-require
+export const AmpImg = require('./index.amp');
```

Comment:
This isn't working in conjunction with  `import { AmpImg } from '../../components/Figure/Image';`